packages/harpstrata/src/harpstrata/HarpStrata/testResources/index.ts

```diff
@@ -9,8 +9,12 @@ const C_MAJOR_DIATONIC_FIRST_POZITION_C_MAJOR_PENTATONIC: HarpStrata = {
   apparatus: MAJOR_DIATONIC_APPARATUS,
   degreeMatrix: EXAMPLE_DEGREE_MATRICES.MAJOR_DIATONIC_FIRST_POZITION,
   pitchMatrix: EXAMPLE_PITCH_MATRICES.MAJOR_DIATONIC_C_HARMONICA,
-  isActiveComplex:
-    EXAMPLE_IS_ACTIVE_COMPLEXES.C_MAJOR_DIATONIC_FIRST_POZITION_C_MAJOR_PENTATONIC,
+  activeDegreeIds:
+    EXAMPLE_IS_ACTIVE_COMPLEXES
```

Comment:
This and it's sources need to be renamed to EXAMPLE_IS_ACTIVE_PAIR or something.

Reply:
Done